### PR TITLE
Patch for Session Event Id

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Sessions/SessionsController.fs [hunk 1]
```diff
@@ -14,15 +14,18 @@ type SessionsController() =
...
 
     let patch (id: Guid) (op: PatchOp) =
-        match op.Path with
+        match op.Path.ToLowerInvariant() with
         | "description" | "title" -> updateField id op.Path op.Value
-        | _ ->  raise <| Exception(sprintf "Error: Patch currently does not accept: %s for session" op.Path) 
-        
+        | "eventid" -> 
+            match Guid.TryParse op.Value with
+            | true, guid -> updateField id op.Path <| Nullable guid
```
Comment (pallenSL) [inline Sessions/SessionsController.fs]: Why "Nullable" guid ?

Comment (jebbinSCL) [inline Sessions/SessionsController.fs]: Adjusted.

**Comment by niksoper** [comment]
:+1:

**Comment by pallenSL** [comment]
:ship:

**Pull request merged**